homeassistant/components/shelly/climate.py

```diff
@@ -308,10 +313,6 @@ async def async_added_to_hass(self) -> None:
 
         self.async_on_remove(self.wrapper.async_add_listener(self._update_callback))
 
-    async def async_update(self) -> None:
-        """Update entity with latest info."""
-        await self.wrapper.async_request_refresh()
-
     @callback
     def _update_callback(self) -> None:
```

Comment:
We can rename this to `_handle_coordinator_update`, which is the default update callback in the parent coordinator entity.
```suggestion
    def _handle_coordinator_update(self) -> None:
```

Reply:
Done